packages/frontend/src/components/common/Filters/FilterInputs/AutoComplete/MultiAutoComplete.tsx

```diff
@@ -114,9 +114,12 @@ const MultiAutoComplete: FC<Props> = ({
                     minimal: true,
                 },
                 inputProps: {
-                    placeholder: disabled
-                        ? undefined
-                        : 'Start typing to filter results',
+                    placeholder:
+                        values.length > 0
+                            ? undefined
+                            : disabled
+                            ? undefined
+                            : 'Start typing to filter results',
```

Comment:
Since both conditions return the same value we could simplify the condition to
```suggestion
                        values.length > 0 || disabled
                            ? undefined
                            : 'Start typing to filter results',
```
Let me know what you think

Reply:
yeah that will perfectly fine.

Reply:
@ZeRego , I made the necessary changes request you to look into it?